tools/bazel

```diff
@@ -0,0 +1,64 @@
+#!/bin/bash
+# Copyright 2019 The gRPC Authors
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+
+# Keeping up with Bazel's breaking changes is currently difficult.
+# This script wraps calling bazel by downloading the currently
+# supported version, and then calling it. This way, we can make sure
+# that running bazel will always get meaningful results, at least
+# until Bazel 1.0 is released.
+# NOTE: This script relies on bazel's feature where //tools/bazel
+# script can be used to hijack "bazel" invocations in given workspace.
+
+set -e
```

Comment:
Consider: `set -x` for debugging purposes.

Reply:
No, please, no.